docs/docs/cmd/planner/roster/roster-add.md

```diff
@@ -0,0 +1,60 @@
+# planner roster add
+
+Creates a new Microsoft Planner Roster
+
+## Usage
+
+```sh
+m365 planner roster add [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+!!! attention
+    The Roster will be automatically deleted when it doesn't contain a plan 24 hours after its creation. Membership information will be completely erased within 30 days of this deletion.
+
+!!! attention
+    This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
+
+!!! important
+    To be able to create a new Roster, the Planner Roster creation tenant setting should be enabled. Point them to our [planner tenant settings list](../tenant/tenant-settings-list.md) command.
```

Comment:
I think this quote was a bit too literally copy pasted 😃. Instead of `Point them to our planner tenant settings list command.` I meant something like "Use planner tenant settings list to check if this setting is enabled for your tenant." or something like that.